coaster/src/lib.rs

```diff
@@ -172,8 +172,9 @@ pub use crate::tensor::{SharedTensor, TensorDesc, ITensorDesc, IntoTensorDesc};
 pub use crate::frameworks::Native;
 #[cfg(feature = "cuda")]
 pub use crate::frameworks::Cuda;
-
+#[cfg(feature = "cuda")]
```

Comment:
at first, I was a bit scared when I saw the dependency, but it really is not that bad - after all the end user wants cuda anyways, so despite the fact `coaster` tries to be agnostics, it makes a lot of sense! :+1: